testgrid/config.yaml

```diff
@@ -5686,6 +5688,11 @@ dashboards:
     test_group_name: pull-aws-ebs-csi-driver-sanity
     description: aws ebs csi driver sanity test
 
+- name: sig-aws-eks-presubmits
+  dashboard_tab:
+  - name: pull-kubernetes-e2e-1-11-aws-eks-1-11-prod-conformance
```

Comment:
IIUC shouldn't the name be `pull-...-e2e-1-10-..-eks-1-11-...` if the test runs with k8s 1.10 e2es against 1.11 eks prod?

Reply:
Ping again~